### PR TITLE
adjust import order

### PR DESCRIPTION
**Pull request by vinicius73**

### CODE REVIEW

File: app/Http/Controllers/Controller.php [hunk 1]
```diff
@@ -3,9 +3,9 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Foundation\Bus\DispatchesJobs;
-use Illuminate\Routing\Controller as BaseController;
 use Illuminate\Foundation\Validation\ValidatesRequests;
 use Illuminate\Foundation\Auth\Access\AuthorizesRequests;
+use Illuminate\Routing\Controller as BaseController;
```
Comment (crynobone) [inline app/Http/Controllers/Controller.php]: Laravel order imports by length.

Comment (vinicius73) [inline app/Http/Controllers/Controller.php]: Did not know. This is an unexpected information for me.

Comment (crynobone) [inline app/Http/Controllers/Controller.php]: https://github.com/laravel/laravel/blob/master/app/Exceptions/Handler.php#L5-L10

Comment (vinicius73) [inline app/Http/Controllers/Controller.php]: Yes I understand.
Just thought unexpected.

**Pull request closed without merging**